pom.xml

```diff
@@ -18,9 +34,9 @@
     </description>
 
     <parent>
-        <groupId>org.sonatype.oss</groupId>
-        <artifactId>oss-parent</artifactId>
-        <version>7</version>
+        <groupId>org.apache</groupId>
```

Comment:
Separate from this PR?  since this pr can be used as a guide for other repos.

Reply:
this has done in previous PR (merge it this will disappear)